src/components/ChatEntry.js

```diff
@@ -1,22 +1,45 @@
-import React from 'react';
 import './ChatEntry.css';
+import TimeStamp from './TimeStamp';
 import PropTypes from 'prop-types';
 
 const ChatEntry = (props) => {
+  const onLiking = () => {
+    const updatedMessage = {
+      id: props.id,
+      sender: props.sender,
+      body: props.body,
+      timeStamp: props.timeStamp,
```

Comment:
hmm you are giving this function a lot of editing power when it only needs to update the `liked` property. Think about giving the function access only to `liked`.